src/Stratis.SmartContracts.CLR.Validation.Tests/SmartContractDeterminismValidatorTests.cs

```diff
@@ -834,6 +834,44 @@ public Test(ISmartContractState state): base(state)
             Assert.False(result.IsValid);
             Assert.NotEmpty(result.Errors);
             Assert.True(result.Errors.All(e => e is ModuleDefinitionValidationResult));
-        }        
+        }
+
+        [Fact]
+        public void SmartContractValidator_Allows_SCL()
+        {
+            var adjustedSource = @"
+using Stratis.SmartContracts;
```

Comment:
Can this be loaded directly from the contract file instead?

Reply:
I find it easier to understand the test if you can see the contract at the same time. I know that's a bit inconsistent across the whole solution.

Reply:
ok